private/protocol/rest/build_test.go

```diff
@@ -20,7 +20,7 @@ func TestCleanPath(t *testing.T) {
 		Scheme: "https",
 		Host:   "host",
 	}
-	cleanPath(uri)
+	cleanPath(uri, "")
```

Comment:
We definitely want a unit test for this behavior.

Reply:
> So it is incorrect to assume that it is safe to alter topic name by removing prefix forward slashes.

I thought I misunderstood how he wants to escape prefix slashes, I'm investigating it in another way